client/public/index.html

```diff
@@ -22,8 +22,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>FoodAdvisor</title>
```

Comment:
I think it would be great for the V2 to use react-helmet so we can have more control